### DISCUSSION TITLE
Should buildx support XDG for ~/.docker/cli-plugins ?

### DISCUSSION THREAD

**Discussion by imme-emosol** [docker/buildx#1826] [General]
Instead of `~/.docker/cli-plugins`,
use `${XDG_CONFIG_HOME:-~/.config}/docker/cli-plugins/`
?

**Comment by crazy-max** [comment]
Should be transferred to [docker/cli](https://github.com/docker/cli) repo. (cc @thaJeztah)

**Comment by imme-emosol** [comment]
:heavy_check_mark: https://github.com/docker/cli/issues/2423
( & some others .. https://github.com/search?q=repo%3Adocker%2Fcli+XDG&type=issues )
